src/test/java/uk/gov/companieshouse/api/accounts/validation/CurrentPeriodValidatorTest.java

```diff
@@ -1,48 +1,151 @@
+
+
+        Errors errors =  validator.validateCurrentPeriod(currentPeriod);
+
+        assertTrue(errors.containsError(
+                new Error("incorrect_total", OTHER_LIABILITIES_OR_ASSETS_NET_CURRENT_ASSETS_PATH,
+                        LocationType.JSON_PATH.getValue(),
+                        ErrorType.VALIDATION.getType())));
+    }
+
+    @Test
+    @DisplayName("ERROR - Other Liabilities Or Assets - Test validation with total assets less current liabilities error")
+    void validateBalanceSheetWithOtherLiabilitiesOrAssetsTotalAssetsLessCurrentLiabilitiesError() {
+        OtherLiabilitiesOrAssets otherLiabilitiesOrAssets = new OtherLiabilitiesOrAssets();
+        otherLiabilitiesOrAssets.setPrepaymentsAndAccruedIncome(4L);
+        otherLiabilitiesOrAssets.setCreditorsDueWithinOneYear(2L);
+        otherLiabilitiesOrAssets.setNetCurrentAssets(2L);
+        otherLiabilitiesOrAssets.setTotalAssetsLessCurrentLiabilities(3L);
+        otherLiabilitiesOrAssets.setCreditorsAfterOneYear(1L);
+        otherLiabilitiesOrAssets.setProvisionForLiabilities(1L);
+        otherLiabilitiesOrAssets.setAccrualsAndDeferredIncome(1L);
+        otherLiabilitiesOrAssets.setTotalNetAssets(2L);
+        balanceSheet.setOtherLiabilitiesOrAssets(otherLiabilitiesOrAssets);
+
+        FixedAssets fixedAssets = new FixedAssets();
+        fixedAssets.setTangible(2L);
+        fixedAssets.setTotalFixedAssets(2L);
+        balanceSheet.setFixedAssets(fixedAssets);
+
+        currentPeriod.setBalanceSheet(balanceSheet);
+        ReflectionTestUtils.setField(validator, "incorrectTotal", "incorrect_total");
+
+        Errors errors =  validator.validateCurrentPeriod(currentPeriod);
+
+        assertTrue(errors.containsError(
+                new Error("incorrect_total", OTHER_LIABILITIES_OR_ASSETS_TOTAL_ASSETS_LESS_CURRENT_LIABILITIES_PATH,
+                        LocationType.JSON_PATH.getValue(),
+                        ErrorType.VALIDATION.getType())));
+    }
+
+    @Test
+    @DisplayName("ERROR - Other Liabilities Or Assets - Test validation with total net assets error")
+    void validateBalanceSheetWithOtherLiabilitiesOrAssetsTotalNetAssetsError() {
+        OtherLiabilitiesOrAssets otherLiabilitiesOrAssets = new OtherLiabilitiesOrAssets();
+        otherLiabilitiesOrAssets.setPrepaymentsAndAccruedIncome(4L);
+        otherLiabilitiesOrAssets.setCreditorsDueWithinOneYear(2L);
+        otherLiabilitiesOrAssets.setNetCurrentAssets(2L);
+        otherLiabilitiesOrAssets.setTotalAssetsLessCurrentLiabilities(2L);
+        balanceSheet.setOtherLiabilitiesOrAssets(otherLiabilitiesOrAssets);
+
+        FixedAssets fixedAssets = new FixedAssets();
+        fixedAssets.setTangible(2L);
+        fixedAssets.setTotalFixedAssets(2L);
+        balanceSheet.setFixedAssets(fixedAssets);
+
+        currentPeriod.setBalanceSheet(balanceSheet);
+        ReflectionTestUtils.setField(validator, "incorrectTotal", "incorrect_total");
```

Comment:
I don't feel like we should have to use reflection in unit tests, but since I don't have an alternative suggestion I'll tentatively approve